docs/install/tutorials/provision.rst

```diff
@@ -13,7 +13,10 @@ You will need an **active Internet connection only to preload content channels o
 Required hardware
 -----------------
 
-#. Laptops and/or other devices to host Kolibri servers.
+#. Laptops and/or other devices to host Kolibri servers. 
+
+	.. tip:: If you are planning to use Raspberry Pi as your server device, read our comprehensive tutorial about setting up :ref:`tutorial_rpi`.
```

Comment:
we might want to wait to advertise this until #104 is addressed

Reply:
`kolibri-server` isn't a seemless addition to the tutorial, that is true.

Reply:
But RPi can certainly have the vanilla Kolibri running on it, it just won't be able to make the best use of its resources, correct?

In other words, if the RPi tutorial is _still_ valid as such (but we should update it as soon as possible), I don't see why would this be a blocker to this PR... :confused:

Reply:
My concern would be the level of advertisement: It's hard to move people from the unchanged RPI tutorial to a new one because the current one needs some adaptation to include `kolibri-server`.

How about this: I fix this during this week, and then we don't worry about the level of advertisement?

Reply:
Ok, valid concern. :+1: 

@indirectlylit should we wait for @benjaoming to fix the RPi tutorial, and then release the updates for the docs, or it's better to hold on (revert) the tutorial related changes and release the docs without them until the fix is in?

Reply:
I think if you just comment out this one line, we can merge the changes

Reply:
Done and ready @indirectlylit! 🙂